static/js/services/turnsettings.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Spreed WebRTC.
+ * Copyright (C) 2013-2015 struktur AG
+ *
+ * This file is part of Spreed WebRTC.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+define(["angular"], function(angular) {
+	"use strict";
+
+	/**
+	 * turnSettings
+	 * Service to correctly sync client and server side TURN settings to contraints.js
+	 * See contraints.js for actual TURN configuration in use for a connection
+	 */
+	return ['mediaStream', 'appData', 'constraints', function(mediaStream, appData, constraints) {
+
+		var preferClientSideTurnSettings = mediaStream.config.TurnClientSideSetable;
+		var turnConfigServer = null;
+
+		/**
```

Comment:
Please remove, we do not use any kind of comment format at the moment, and i would prefer to keep this open until we decide to use something.